SU-HSDO/suhumsci
# Added config_import_de for local and dev environment

# READY FOR REVIEW

# Summary
- Delete custom entities before config gets imported.
- Prevents errors when trying to clone a site that might have custom content types, custom taxonomy vocabularies, etc.

# Needed By (Date)
- Whenever

# Urgency
- Low

# Steps to Test

1. checkout `develop` branch
1. try to clone francestanford `blt drupal:sync --site=francestanford`
1. Notice all the errors during the config import portion
1. checkout this branch
1. `composer update --prefer-source`
1. clone france again `blt drupal:sync --site=francestanford --sync-files`
1. validate no errors during clone process
1. clone with partial `blt drupal:sync --site=francestanford --partial`
1. validate the custom content type "Conference Overview" and the content with that still exist


# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
